src/views/CompanySettings.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from "react-redux";
+import { fetchSingleHotel, updateHotel } from '../store/actions/hotel';
+
+import SuperAdminNav from '../components/SuperAdminNav';
+
+class CompanySettings extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.props = props;
+    const { state, dispatchFetchSingleHotel } = this.props;
+    this.state = { companyName: state.hotel.name, companyMotto: state.hotel.motto };
```

Comment:
This should be on global state store. And from there connected to this component in `mapStateToProps`